### PR TITLE
FEA-133 Change city page route

### PR DESCRIPTION
**Pull request by darya-karpovich-cc**

### CODE REVIEW

**Review by adrian-konopnicki-CC — APPROVED** [review]

File: src/utils/convertSpecialCharacters/convertSpecialCharacters.test.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import {convertSpecialCharacters} from './convertSpecialCharacters';
+
+describe('convertSpecialCharacters', () => {
+
+  it('should replace spacial characters', () => {
+    
+    expect(convertSpecialCharacters('ąęćńóśźżł')).toBe('aecnoszzl');
```
Comment (kamil-borecki-cc) [inline src/utils/convertSpecialCharacters/convertSpecialCharacters.test.ts]: **praise** nice! but we are still missing some of polish diacritics :)

Comment (klaudia-blazyczek-cc) [reply src/utils/convertSpecialCharacters/convertSpecialCharacters.test.ts]: **question** should we also check letters like ä etc?

Comment (darya-karpovich-cc) [reply src/utils/convertSpecialCharacters/convertSpecialCharacters.test.ts]: @kamil-borecki-cc  We added capital letters, is there anything else we missed?

Comment (kamil-borecki-cc) [reply src/utils/convertSpecialCharacters/convertSpecialCharacters.test.ts]: @darya-karpovich-cc sorry i misread this and i thought we are missing żź :D

**Review by kamil-borecki-cc — APPROVED** [review]

**Pull request merged**